packages/components/src/locale/en-US.json

```diff
@@ -149,6 +149,7 @@
   "badge__backed_up": "Backed up",
   "badge__coming_soon": "Coming soon",
   "checkbox__manual_backup_warning": "I understand that if I lose my recovery phrase, I will lose my funds.",
+  "content__account_lowercase": "",
```

Comment:
Translation for this is missing. @loaddss

Reply:
Thanks